cmd/parca-agent/main.go

```diff
@@ -102,71 +104,50 @@ func main() {
 	ctx := context.Background()
 	var g run.Group
 
-	var (
-		err error
-		wc  profilestorepb.ProfileStoreServiceClient = agent.NewNoopProfileStoreClient()
-		dc  debuginfo.Client                         = debuginfo.NewNoopClient()
-	)
+	var wc profilestorepb.ProfileStoreServiceClient = agent.NewNoopProfileStoreClient()
+	var dc debuginfo.Client = debuginfo.NewNoopClient()
 
 	if len(flags.StoreAddress) > 0 {
-		conn, err := grpcConn(reg, flags)
+		_, err := grpcConn(reg, flags)
 		if err != nil {
 			level.Error(logger).Log("err", err)
 			os.Exit(1)
 		}
-
-		wc = profilestorepb.NewProfileStoreServiceClient(conn)
```

Comment:
whops I think this was removed accidentally, without this we won't be sending anything to a Parca server

Reply:
right, good catch!